src/View/Helper/FormHelper.php

```diff
@@ -2001,6 +2001,26 @@ public function postLink(string $title, array|string|null $url = null, array $op
         return $out;
     }
 
+    /**
+     * Creates an HTML link, but access the URL using the method you specify
+     *  (defaults to DELETE). Requires javascript to be enabled in browser.
```

Comment:
```suggestion
     * Creates an HTML link, that submits a form to the given URL using the DELETE method.
     *  Requires javascript to be enabled in browser.
```

The HTTP method cannot be changed in this method.